src/components/features/Brands/Brands.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { getBrands } from '../../../redux/brandsRedux';
+
+import styles from './Brands.module.scss';
+
+const Brands = () => {
+  const brands = useSelector(state => getBrands(state));
+  return (
+    <div className='container mt-3 mb-5'>
+      <div className={`border-top border-bottom ${styles.brands}`}>
+        <div className={`border m-3 ${styles.arrow}`}>
+          <span className='text-muted'>{'<'}</span>
+        </div>
+        <div className='row d-flex w-100 m-3 text-center'>
+          {[1, 2, 3, 4, 5, 6].map(item => (
```

Comment:
A dlaczego nie zrobisz pętli po 'bfands'. Skoda wiesz ze zawsze będzie 6 brandów?

A no i index w tablicy zaczyna się od 0

Reply:
Task był na wygląd komponentu zgodnie z mockup'em więc 6 brandów a w storze dodałem ich na przyszłość już więcej bo wiedziałem, że będzie przesuwanie. Chciałem zrobić teraz tak zgodnie z taskiem a potem przy tasku który jest na funkcjonalność ogarnąć ostatecznie tą pętlę i przesuwanie. Jeśli chcesz mogę zrobić całość od razu ;)

Reply:
Przesuwanie to swoją drogą, ale jak już jest to w storze to sposób wyświetlania zróbmy zgodnie ze sztuką, nie musi być już pod przesuwanie robione, no ale jak masz tablice elementów to jedynym wyjściem jest pętla po tych elementach a nie pętla po tablicy z liczbami o długości odpowiadającej ilości elementyw :)

Reply:
Wiesz co, sam się zapędziłem w kozi róg bo dodałem do stanu więcej elementów niż 6 na przyszłość ale bez funkcjonalności i RWD z następnych tasków przy mapowaniu rozjechałoby się wszystko stąd najpierw na sztywno a potem ta tablica ;) Zmieniłem na map i uciąłem store do 6 elementów. Jak dojdzie RWD i funkcjonalność to można tam dorzucać.